pages/Transactions/index.tsx

```diff
@@ -1,8 +1,89 @@
+import * as React from 'react';
 import Layout from "../../components/Layout/Layout";
+import PropTypes from 'prop-types';
+import { Grid, Button, Tabs, Tab, Box } from "@mui/material";
+import styles from './Transactions.module.css';
 
 const TransactionsPage: React.FunctionComponent = () => {
+
+    function TabPanel(props) {
```

Comment:
I think the TabPanel functional component should be moved into the components folder and be defined there instead of within the index file in Transactions. (this will make the component reusable in other pages should that use case come up, modularize our components, and make our code more organized!)